tests/cypress/template/navbar-languages.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('Navbar languages', () => {
+
+  it('Sets and reads SKOSMOS_LANGUAGE cookie correctly', () => {
+    cy.visit('/')
```

Comment:
I think there should be a test here to verify that the URL does *not* include `/fi/`. Then the test would demonstrate that the cookie setting actually had an effect on `/`.

Reply:
Good point!